run_sudo_unit_tests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
```

Comment:
Why does this need to be run as root? Isn't the point of using "sudo" that this is not required? If this is for testing why not setup a proper sudoers file?